src/components/DropdownMenu/DropdownMenu.js

```diff
@@ -11,6 +11,29 @@ export default class DropdownMenu extends React.Component {
 
   someDescription;
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      active: this.props.active,
+    };
+  }
+
+  _onClickOutside = (e) => {
```

Comment:
Detection of wether or not the clicked happened `outside` is done in the method.
```suggestion
  _onClick = (e) => {
```